### DISCUSSION TITLE
🖋️: トラブルシュートに情報が溜まっていくような仕組みが必要なのでは？

### DISCUSSION THREAD

**Discussion by tani-takanori** [ws-4020/mobile-app-crib-notes#115] [Ideas]
StackOverFlowのようにQとAが溜まっていくような仕組みがあったほうがトラブルシュートに情報が集まるのでは？

**Comment by tani-takanori** [comment]
トラブルシュートはStackOverFlowを目指しているわけではない。

**Reply by tani-takanori** [reply]
StackOverFlowなどのQAサイトをサルベージして利用頻度、参照頻度が高いものをガイドしてもいいかもしれない

**Reply by ghost** [reply]
> StackOverFlowなどのQAサイトをサルベージして利用頻度、参照頻度が高いものをガイドしてもいいかもしれない

「トラブルシュート」のページを用意するのではなく、自分たちのところに来た質問などからよくハマりがちな罠を具体的に検知して、それらをまとめたページとする方が良いのでは、と思います。なお現時点での対応は、「トラブルシュート」というページ名を変更するのみで充分と考えています。

**トラブルシュート**は各々がトラブルに陥った時に、React NativeとExpoの公式サイトやIssue、ソースコードを自分たちで確認してもらうのが良いかと。コンテンツはトラブルを起こさずに進めやすくするためのものと考えているので、そもそも「トラブルシュート」という考え方があまり相応しくなかったのかな、とも思いました。

このIdeaからは少し話題がそれますが、トラブルシュートの際に誤った情報にたどり着きがちなのでそれをどうやって回避するかは検討したほうが良いと思います。特に、環境構築周りで犠牲者が目立ちます。

**Comment by ghost** [comment]
QAをDiscussionでためておくのはどうだろうかと思い、いくつか転記してみました。
でも、これを突き詰めていくとみんなにStack Overflow（or 社内の情報共有システム）で質問してもらえばいいという話になりそうな気がしてきました。

* #111
* #112
* #121

**Comment by tani-takanori** [comment]
https://github.com/ws-4020/mobile-app-crib-notes/pull/237

**Comment by creature-water-valley** [comment]
- #237 で「ハマりがちな落とし穴」に変更
- #1266 で「トラブルシューティング」に変更
  - トラブルシューティング関連のコンテンツしか追加されなかったため
  - 「ハマりがちな落とし穴」だと英語表記がわかりづらく、そのページを表示する人が少ないそうと思ったため
